### PR TITLE
Avoid wireAttributes() in Client

### PR DESCRIPTION
**Pull request by t-wissmann**
Also, group attributes together (even though some of them are not
attributes in winterbreeze yet, but only in master)

### CODE REVIEW

File: src/client.h [hunk 1]
```diff
@@ -51,6 +39,21 @@ class Client : public Object {
...
+
+    // attributes:
+    Attribute_<bool> urgent_;
+    Attribute_<bool> fullscreen_;
+    Attribute_<std::string> title_;  // or also called window title; this is never NULL
+    Attribute_<std::string> window_id_str;
+
+    Attribute_<std::string> keyMask_; // keymask applied to mask out keybindins
+    Attribute_<bool> pseudotile_; // only move client but don't resize (if possible)
+    bool        ewmhrequests_ = true; // accept ewmh-requests for this client
```
Comment (dnnr) [inline src/client.h]: Why these spaces after `bool`?

Comment (t-wissmann) [reply src/client.h]: I've only reordered lines and this was from a previous alignment.

Comment (dnnr) [reply src/client.h]: Ah! I'd suggest cleaning that up while at it. It wouldn't make the diff any harder to read, imho.

Comment (t-wissmann) [reply src/client.h]: Ah, I've just noticed the "(keep)" in your question.
These lines will eventually be replaced by `Attribute_<bool>`, so let us please not worry about the spaces for the Moment.

Comment (dnnr) [reply src/client.h]: Alright, nevermind then.

**Pull request merged**